### ISSUE TITLE
PHP error: implode(): Argument #2 ($array) must be of type ?array, string given

### ISSUE THREAD

**Issue by lourdas** [backdrop-ops/localization#48]
While translating Backdrop for the Greek language, I got this error:

![Screenshot_20240211_093954](https://github.com/backdrop-ops/localize.backdropcms.org/assets/397916/fbec018c-eedd-4f57-8f49-cc9cf614c0e5)

I got this error, while using the filter conditions:

![Screenshot_20240211_094331](https://github.com/backdrop-ops/localize.backdropcms.org/assets/397916/4d4dc0cf-4d71-4db2-991b-22b359ab8c7b)

There are a couple of terms that have singular and plural in this page, in case this helps.

**Comment by argiepiano** [comment]
@lourdas thanks for reporting. For the record, the full error is:

<img width="915" alt="Screen Shot 2024-02-11 at 7 39 18 AM" src="https://github.com/backdrop-ops/localize.backdropcms.org/assets/9938978/5bb2d187-ff13-427c-812b-b52ebf01f818">

**Comment by argiepiano** [comment]
@lourdas I"m trying to reproduce. I can't set the filter "Release". For me there is no 1.27 option there. Can you provide more details about how you got to that page?

<img width="1197" alt="Screen Shot 2024-02-11 at 7 47 07 AM" src="https://github.com/backdrop-ops/localize.backdropcms.org/assets/9938978/87dd281b-433a-4399-ad15-c95e499aaa76">

**Comment by lourdas** [comment]
You have to select the Greek language, then click on Translate, Reveal more filters, Project -> Backdrop CMS, Release -> 1.27, Status -> Untranslated and click on Filter.

**Comment by argiepiano** [comment]
OK, I was able to get the 1.27 filter. I'm looking at the Greek translations. I see only 413 pages - your screenshot shows you are on page 439 - are you looking at Greek translations?

<img width="1265" alt="Screen Shot 2024-02-11 at 7 51 46 AM" src="https://github.com/backdrop-ops/localize.backdropcms.org/assets/9938978/4ab79309-49f8-4f2f-b076-b31bcf8d6946">

**Comment by lourdas** [comment]
Yeah, I'm in the process on translating right now, so, I guess the page is not valid anymore, the untranslated pages are less now. Let me check if I can find this specific page.

**Comment by argiepiano** [comment]
I'm looking at the code that produced the error. It's called only twice, from the submit handler for the translation form (`l10n_community_translate_form_submit()`), and from a function that creates a form segment for translated strings (`_l10n_community_translate_translation()`) So, I suspect this is happening in the second one, when the site tries to build the form for a specific string.

**Comment by lourdas** [comment]
I don't seem to find the page, sorry. I've translated a lot of strings since this morning and it happened only at that specific page when I created the issue. If it happens again, I will let you know here, so at the moment please leave this open.

**Comment by argiepiano** [comment]
Sounds good. Thanks
